Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml

```diff
@@ -151,8 +167,10 @@ tasks:
       brand: VxStream
     nexttasks:
       '#none#':
-      - "16"
+      - "17"
     scriptarguments:
+      dontThrowErrorOnFileDetonation:
```

Comment:
this should be an input to the playbook with `false` as default

Reply:
the command default is false